packages/rancher-cis-benchmark/charts/README.md

```diff
@@ -0,0 +1,14 @@
+# Rancher CIS Benchmark Chart
+
+A Rancher created chart that packages the rancher/cis-operator binary to install via a helm chart.
+
+# Installation
+
+### Requirements
+
+This chart depends on the rancher-cis-benchmark-crd chart.
+
+### Installation
+```
+helm install rancher-cis-benchmark ./ --create-namespace -n cis-operator-system
```

Comment:
Nit: add in CRD chart install here as well.

Reply:
There is no separate CRD chart - its all in the single `crds` folder now. So I dont think there is a CRD chart install command needed - correct?

Reply:
For manual installs there is assuming they are using the published version. If they use the vanilla git version it's all one command.